packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -87,7 +87,7 @@ export const sortings = {
  * ~ $lt: before.endOf('day').
  */
 Posts.addDefaultView(terms => {
-  const validFields = _.pick(terms, 'userId', 'meta', 'groupId', 'af','question', 'authorIsUnreviewed');
+  const validFields = _.pick(terms, 'userId', 'meta', 'af','question');
```

Comment:
Why are you doing this? Seems like it would break group pages?

Reply:
I'm doing this because lint or something was complaining about it being a duplicate , and I was a bit confused about that, and mostly figured if I included it in the PR someone would say "what?" and then we'd remember to have a conversation about it.

I think looking at it now I realize that we do want it to exist with a default which can get overwritten. So I guess the problem is probably more that our lint is complaining incorrectly?

Reply:
Yeah, I would do this with ES6 syntax these days, without a reference to `_.pick`, which I think should also handle the duplication problem. Happy to briefly pair on it.